example_test.go

```diff
@@ -86,6 +87,9 @@ func ExampleParseWithClaims_customClaimsType() {
 	token, err := jwt.ParseWithClaims(tokenString, &MyCustomClaims{}, func(token *jwt.Token) (interface{}, error) {
 		return []byte("AllYourBase"), nil
 	})
+	if err != nil {
```

Comment:
The `fatal` function is available here, and elsewhere, but without context it may be confusing, so I just expanded it.